frontends/web/src/routes/exchange/components/sell.tsx

```diff
@@ -0,0 +1,217 @@
+import { useNavigate } from 'react-router-dom';
```

Comment:
This component has many duplicated code from `exchange.tsx` and needs to be refactored (both this and `exchange.tsx`). But not doing it now as it'd be a refactoring PR instead.